types/node/crypto.webcrypto.d.ts

```diff
@@ -0,0 +1,20 @@
+declare module "crypto" {
+    class SubtleCrypto {
+        generateKey(...args: any[]): Promise<string>;
+    }
+    class CryptoKey {}
+
+    /** Basic cryptography features available in the current context. It allows access to a cryptographically strong random number generator and to cryptographic primitives. */
+    interface Crypto {
+        readonly subtle: SubtleCrypto;
+        getRandomValues<T extends Int8Array | Int16Array | Int32Array | Uint8Array | Uint16Array | Uint32Array | Uint8ClampedArray | Float32Array | Float64Array | DataView | null>(array: T): T;
+    }
+    interface NodeCrypto extends Crypto {
+        readonly CryptoKey: typeof CryptoKey;
+    }
```

Comment:
This is a very incomplete webcrypto interface. Any reason not to ref lib dom and just extend that?

Reply:
> Any reason not to ref lib dom and just extend that?

The original PR had raise some errors, you can see from here https://github.com/DefinitelyTyped/DefinitelyTyped/pull/48981/checks?check_run_id=1359476159:

> 2> Error: Errors in typescript@4.2 for external dependencies:
Error: 2> ../node/crypto.webcrypto.d.ts(2,34): error TS2304: Cannot find name 'Crypto'.
Error: 2> ../node/crypto.webcrypto.d.ts(3,36): error TS2304: Cannot find name 'CryptoKey'.
Error: 2> ../node/events.d.ts(81,31): error TS2304: Cannot find name 'Event'.
Error: 2> ../node/events.d.ts(82,37): error TS2304: Cannot find name 'EventTarget'.
Error: 2> ../node/globals.d.ts(473,33): error TS2304: Cannot find name 'AbortController'.
Error: 2> ../node/globals.d.ts(474,29): error TS2304: Cannot find name 'AbortSignal'.
Error: 2> ../node/net.quic.d.ts(112,18): error TS2304: Cannot find name 'AbortSignal'.
Error: 2> ../node/timers.d.ts(12,18): error TS2304: Cannot find name 'AbortSignal'.
Error: 2> ../node/worker_threads.d.ts(243,38): error TS2304: Cannot find name 'MessageEvent'.
2> 
2>     at /home/runner/work/DefinitelyTyped/DefinitelyTyped/node_modules/dtslint/bin/index.js:206:19
2>     at Generator.next (<anonymous>)
2>     at fulfilled (/home/runner/work/DefinitelyTyped/DefinitelyTyped/node_modules/dtslint/bin/index.js:6:58)


BTW, I didn't have so much time on continue working on this PR, it would be appreciate if someone can pick it up.